ldi-core/sparql-construct/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/SparqlConstructTransformer.java

```diff
@@ -70,4 +73,11 @@ private List<Model> splitDataset(Dataset dataset) {
 		return result;
 	}
 
+	private void initGeoFunctions() {
```

Comment:
Can you add a unit test that verifies that these are initialized? (I am guessing you can test this by calling the FunctionRegistry with "FirstCoordinate.name", etc. and check if they exist)

Reply:
I noticed a method 'isRegistered', perhaps you can use that one in the test